docs/environments/pendulum.md

```diff
@@ -1,4 +1,56 @@
 # Pendulum
+| Overview          | Specification                                  |
+|-------------------|------------------------------------------------|
+| Observation Space | 3-d array: [cos(theta), sin(theta), theta_dot] |
```

Comment:
The other environments document the ranges as well. cos and sin have implied ranges, but theta_dot doesn't.

Checking the code, this could be infinite? Or maybe the friction effectively limits it, but we don't know where?

Reply:
I actually wanted to know the bounds on the angular velocity myself for some purpose. The default bound (`_MAX_SPEED`) is `np.inf`. Empirically I've not seen it exceed 8. Perhaps one could analytically compute the maximum velocity using the differential equation of motion. 

For now I'll say that by default there is no bound on it?